src/client/snowpack.config.js

```diff
@@ -2,4 +2,10 @@ module.exports = {
   workspaceRoot: false,
   testOptions: { files: ["**/tests/**/*", "**/*.test.*"] },
   buildOptions: { out: "../idom/client" },
+  mount: { public: { url: "/", static: true } },
+  optimize: { bundle: true, target: "es2018" },
+  alias: {
+    react: "preact/compat",
+    "react-dom": "preact/compat",
```

Comment:
I copy-pasted this from the snowpack docs I think, so I'd imagine so.

Reply:
I didn't realize this was legal JS object syntax until I just tested it now. Kinda sketches me out a bit.

[Snowpack alias docs](https://www.snowpack.dev/reference/configuration#alias) don't seem to use quotes.

Reply:
Seems like quotes can be used if the name contains typically illegal characters (such as "@"). Since "react-idom" is all standard characters we don't need that here.

Reply:
Quotes are required because of the `-`.